documentation/modules/rn-2.2.adoc

```diff
@@ -9,21 +9,33 @@ You can migrate virtual machines (VMs) from VMware vSphere or {rhv-full} to {vir
 
 The release notes describe new features and enhancements, known issues, and technical changes.
 
-// [id="technical-changes-22_{context}"]
-// == Technical changes
-//
-// .TBD
-//
-// TBD
-//
-// [id="new-features-and-enhancements-22_{context}"]
-// == New features and enhancements
-//
-// This release adds the following features and improvements.
-//
-// .TBD
-//
-// TBD
+[id="technical-changes-22_{context}"]
+== Technical changes
+
+.Setting interval between precopies
+
+You can set the precopy interval.
+
+.Handling RHV VMs with USB enabled
+
+You can migrate an RHV VM with a host device of the USB type mapped to it, although the mapping is not supported on the target side.
+
+[id="new-features-and-enhancements-22_{context}"]
+== New features and enhancements
+
+This release adds the following features and improvements.
+
+.Downloading must-gather logs from the web console
```

Comment:
Remove "must-gather". I am fairly certain that the UI does not mention must-gather. The button says "Get logs".